js/notify/electron-notify.js

```diff
@@ -208,12 +227,32 @@ function calcDimensions() {
 function setupConfig() {
     closeAll();
 
-    // Use primary display only
-    let display = electron.screen.getPrimaryDisplay();
+    // This feature only applies to windows
+    if (!isMac) {
+        let screens = electron.screen.getAllDisplays();
+        if (screens && screens.length >= 0) {
+            externalDisplay = screens.find((screen) => {
+                let screenId = screen.id.toString();
+                return ((screen.bounds.x !== 0 || screen.bounds.y !== 0) && screenId === notfDisplay);
+            });
+        }
+    }
 
-    config.corner = {};
-    config.corner.x = display.bounds.x + display.workArea.x;
-    config.corner.y = display.bounds.y + display.workArea.y;
+    let display;
+    // Update the notification to display on the user selected screen
+    if (externalDisplay) {
+        // use user selected display
+        display = externalDisplay;
+        config.corner = {};
+        config.corner.x = display.bounds.x;
+        config.corner.y = display.bounds.y;
+    } else {
+        // Use primary display
+        display = electron.screen.getPrimaryDisplay();
+        config.corner = {};
+        config.corner.x = display.bounds.x + display.workArea.x;
```

Comment:
For external display we should not be adding the ```display.workArea``` because the ```display.workArea.x``` value includes the width of the ```PrimaryDisplay```.

> **Primary Display**
bounds: { x: 0, y: 0, width: 1280, height: 800 }
workArea: { x: 0, y: 23, width: 1280, height: 737 }
workAreaSize: { width: 1280, height: 737 }

> **External Display**
bounds: { x: 1280, y: 0, width: 1920, height: 1080 }
workArea: { x: 1280, y: 23, width: 1920, height: 1057 }
workAreaSize: { width: 1920, height: 1057 }

As you can see if we add ```display.workArea``` the notification window's ```corner.x``` value would be ```bounds : 1280 + workArea : 1280```.

So i think this code cannot be simplified because the external display should be handled separately.